bindings/twilio/sendgrid/sendgrid.go

```diff
@@ -31,10 +31,16 @@ import (
 	kitmd "github.com/dapr/kit/metadata"
 )
 
+const (
+	keyAsmGroupID         = "asmGroupId"
+	keyAsmGroupsToDisplay = "asmGroupsToDisplay"
```

Comment:
I could replace the other raw strings in this package with constants but I wanted to save any of that for a separate PR to make reviewing changes easier